src/game/RandomizationOfShipsPositions.js

```diff
@@ -0,0 +1,85 @@
+import { SHIPS_LIST } from "./SHIPS_LIST";
+
+export const gameBoardArray = [];
+let direction;
+let Position;
+const GAME_BOARD_WIDTH = 10;
+
+const createGameBoardArray = () => {
+  for (let i = 0; i < GAME_BOARD_WIDTH; i++) {
+    gameBoardArray.push([]);
+    for (let j = 0; j < GAME_BOARD_WIDTH; j++) {
+      gameBoardArray[i].push(0);
+    }
+  }
+};
+createGameBoardArray();
+
+const directionRoll = () => {
+  Math.floor(Math.random() * 2) === 0
+    ? (direction = "vertical")
+    : (direction = "horizontal");
+};
+
+const makePosition = (k) => {
```

Comment:
An argument called `k` doesn't give anyone any information about what it is. Call it in a human understandable way.